.github/workflows/dockerimage.yml

```diff
@@ -27,13 +27,13 @@ jobs:
 
     runs-on: ubuntu-latest
 
-    # TODO (crcrpar): Remove `exclude` once integration works with Python 3.9.
+    # TODO (nzw0301): Remove `exclude` once integration works with Python 3.10.
     strategy:
       matrix:
-        python_version: ['3.6', '3.7', '3.8', '3.9']
+        python_version: ['3.6', '3.7', '3.8', '3.9', '3.10']
         build_type: ['', 'dev']  # "dev" installs all the dependencies including pytest.
         exclude:
-        - python_version: '3.9'
+        - python_version: '3.10'
           build_type: 'dev'
```

Comment:
Python 3.9-dev image will be newly published. I don't think it problematic because of CI results.